WcaOnRails/app/models/competition.rb

```diff
@@ -513,7 +511,7 @@ def end_date
     if @start_date.blank?
       self.year = self.month = self.day = 0
     else
-      unless /\A\d{4}-\d{2}-\d{2}\z/.match(@start_date)
+      unless /\A\d{4}-\d{2}-\d{2}\z/ =~ @start_date
```

Comment:
I'm okay turning this one off, it feels pretty silly to me. This is filed under [`Performance/RedundantMatch`](http://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Performance/RedundantMatch). I guess the idea is that `.match` does more work than `=~` building up some object that we don't really use. This feels like quite a microoptimization however.

I'll wait for you to respond before doing anything here, but I personally am inclined to turn off `Performance/RedundantMatch`.